scout/dashboard.md

```diff
@@ -41,12 +41,33 @@ The repository tag list shows all tags for a repository. You can search for spec
 
 Each entry in the list shows the following details:
 
+- A checkbox to mark the tag for comparison to one other.
+
+  > **Tip**
+  >
+  > Compare two tag images by selecting the checkboxes next to them and selecting the **Compare images** button at the top of the list.
+  {: .tip }
+
 - The tag version. Clicking the link for version opens [the image layer view](#image-layer-view).
 - The operating system and architecture of the image.
 - The vulnerabilities for the tag version.
 - The last push for the tag version.
 - The base image and version used by the repository and the vulnerabilities for
   that version.
+- The size of the tag image.
+
+#### Comparing two tag images
+
+![Screenshot of an overview of differences between two selected images](./images/dashboard-diff-top.png)
+
+The top section of the comparison view shows an overview of the two selected image tags.
+
+![Screenshot of overview differences between two selected images](./images/dashboard-diff-bottom.png)
+
+The tabs section of the view shows the following:
+
+- Select the **Packages** tab to see packages added, removed, or updated in each image. Each entry in the table shows the differences between the versions and vulnerabilities in each image. Select the disclosure triangle next to a package to see more detail on the vulnerabilities changed.
```

Comment:
On the screen we use the terminology "changed" instead of "updated", this is because it could be a downgrade, an upgrade, or a sideways move, and it's applying to both images not just the final image. Should we use the same terminology in both places? (The UI and the docs).